crates/re_space_view_spatial/src/space_view_2d.rs

```diff
@@ -248,7 +249,22 @@ impl SpaceViewClass for SpatialSpaceView2D {
             .show(ui, |ui| {
                 state.default_size_ui(ctx, ui);
                 state.bounding_box_ui(ctx, ui, SpatialSpaceViewKind::TwoD);
-                ui.end_row();
+
+                if let Some(bounds) = state.state_2d.bounds {
+                    ctx.re_ui
+                        .grid_left_hand_label(ui, "Bounds")
+                        .on_hover_text("The currently visible area.");
```

Comment:
Technically more than this is visible, so I wonder how we could formulate this more accurately.
How about:

```suggestion
                        .on_hover_text("The area guaranteed to be visible.\nDepending on the view's current aspect ratio the actually visible area might be larger either horizontally or vertically.");
```